### PR TITLE
[IMP] added new way to apply patches

### PR DESCRIPTION
**Pull request by shepilov-vladislav**
… been maintained.

### CODE REVIEW

**Comment by blaggacao** [comment]
@shepilov-vladislav Should we dig deeper on this one? @ychirino  is trying to integrate into a development workflow a case where only individual patches should be applied.

**Comment by shepilov-vladislav** [comment]
> @shepilov-vladislav Should we dig deeper on this one? @ychirino is trying to integrate into a development workflow a case where only individual patches should be applied.

@blaggacao I think we can split patches to folder `/patch.d/...` for example. In this way we can apply only required patches on build/re-build/inherit build/.. stages. What do you think about it? And what do you think how we can add some tests for docker images? Maybe you know some approaches?

**Comment by blaggacao** [comment]
@shepilov-vladislav patch.d looks just like the addition we might looking for to make this patch ecosystem more versatile. The only backdraw I see, we have to remove these lines:https://github.com/xoe-labs/dockery-odoo-scaffold/blob/c1c7b2f7961f0030d22e0b9ca6e6b90b790d1130/docker-compose.yml#L23-L25 in order to let the patches effectuate. I guess this is a good compromise, because it let's the user choice:
- Either rebuilding when updating odoo and NOT debugging odoo itself OR
- maintaining a synthetic branch of odoo with applied patches (more fleixible for the more experienced developer).

File: v-11.0/out/patches [hunk 1]
```diff
@@ -4,39 +4,31 @@
...
+                ./patch.py --directory=$1 $line
+                echo $line
+            done < "$patches_file"
+    else
+        echo "Not found file: $patches_file"
+        cat $patches_file
+    fi
+}
+
+array=("/patches.txt")
```
Comment (blaggacao) [inline v-11.0/out/patches]: @shepilov-vladislav So it would be to replace this with a dir-walker, instead? Sounds good!

**Comment by blaggacao** [comment]
emented wholly differently.

**Pull request closed without merging**